RELEASE.md

```diff
@@ -0,0 +1,27 @@
+# Release History
+
+## Table of Contents
+
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [Latest Release Information](#latest-release-information)
+    - [2023-12-12](#2023-12-12)
+
+## Acronyms, Terms and Abbreviations
+
+- `HAL` - Hardware Abstraction Layer
+- `RDK-V` - Reference Design Kit - Video
+
+## Latest Release Information
```

Comment:
What happened to all the other good paragraphs that where in this doc?

One for the jump point for the HPK Understanding ( and that's just a draft version with some paragraphs, in it, with links to other important documentation ).


Then a completely seperate doc, than can be autoappended and auto generated too which contains the release information.

Reply:
the README.md will be hugely expanded over time and will contain the reference HPK information for all the HAL's.